lib/hedwig_mopidy/responders/mopidy.ex

```diff
@@ -238,6 +239,13 @@ defmodule HedwigMopidy.Responders.Mopidy do
     send message, response
   end
 
+  def add_tracks_in_batches(tracks, batch_size \\ 100) do
+    tracks
+    |> Enum.chunk(batch_size, batch_size, [])
```

Comment:
unfortunately not, at least as I read the docs: https://hexdocs.pm/elixir/Enum.html#chunk/4

if you want to specify `leftover` you need the 3rd parameter which doesn't have a default value in the /4 case. Kind of awkward.